packages/chart/src/components/LineChart/helpers.ts

```diff
@@ -92,38 +125,44 @@ export const filterCircles = (
 }
 
 const isCalculable = value => !isNaN(parseFloat(value)) && isFinite(value)
-const handleFirstIndex = (data, seriesKey, preliminaryData) => {
+const handleFirstIndex = ({
+  data,
+  seriesKey,
+  preliminaryData,
+  dynamicCategory,
+  originalSeriesKey,
+  colorScale,
+  isSuppressed
+}) => {
   let pairCount = '0'
   const result = {
     data: { '0': [] },
-    style: ''
+    style: '',
+    color: ''
   }
 
   // If data is empty, return the empty result
   if (!data.length) return result
 
   const firstIndexDataItem = data[0]
 
-  // Function to check if a data item matches the suppression criteria
-  const isSuppressed = pd => {
-    if (pd.type === 'effect' || pd.hideLineStyle) return
-    return (
-      pd.type == 'suppression' && pd.value === firstIndexDataItem[seriesKey] && (!pd.column || pd.column === seriesKey)
-    )
-  }
-
   // Find applicable suppression data for the first item
-  const suppressionData = preliminaryData.find(isSuppressed)
+  const suppressionData = preliminaryData.find(item => isSuppressed(item, firstIndexDataItem))
```

Comment:
Check for preliminaryData before running any functions on it.

Reply:
Fixed!